homeassistant/components/generic/camera.py

```diff
@@ -95,8 +95,10 @@ def __init__(
             self._still_image_url = Template(self._still_image_url, hass)
         self._stream_source = device_info.get(CONF_STREAM_SOURCE)
         if self._stream_source:
-            self._stream_source = Template(self._stream_source, hass)
-        self._limit_refetch = device_info[CONF_LIMIT_REFETCH_TO_URL_CHANGE]
+            if not isinstance(self._stream_source, Template):
+                self._stream_source = Template(self._stream_source, hass)
```

Comment:
Why is `stream_source` sometimes a template, is it when creating a preview entity?

Reply:
You are right, `stream_source` is always a string at this point, so no need for the check.  Fixed in b2401946a43610da07ced5be5522e62bc6e96df6